flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -138,29 +125,13 @@ public FlutterStudioMonitorStageView(@NotNull FlutterStudioProfilersView profile
     yAxisBytes.setMarkerLengths(MARKER_LENGTH, MARKER_LENGTH);
     yAxisBytes.setMargins(0, Y_AXIS_TOP_MARGIN);
 
-    // TODO(terry): Better Y axis.
-    /*
-    final JPanel axisPanel = new JBPanel(new BorderLayout());
-    axisPanel.setOpaque(false);
-    final AxisComponent memoryAxis = new AxisComponent(getStage().getMemoryAxis(), AxisComponent.AxisOrientation.RIGHT);
-    memoryAxis.setShowAxisLine(false);
-    memoryAxis.setShowMax(true);
-    memoryAxis.setShowUnitAtMax(true);
-    memoryAxis.setHideTickAtMin(true);
-    memoryAxis.setMarkerLengths(MARKER_LENGTH, MARKER_LENGTH);
-    memoryAxis.setMargins(0, Y_AXIS_TOP_MARGIN);
-    axisPanel.add(memoryAxis, BorderLayout.WEST);
-
-    // Then need to add yAxis to topPanel (See below).
-    topPanel.add(yAxis, new TabularLayout.Constraint(0, 1));
-
-    */
-
     LineChartModel model = new LineChartModel();
 
     FlutterAllMemoryData.ThreadSafeData memoryUsedDataSeries = stage.getMemoryUsedDataSeries();
     FlutterAllMemoryData.ThreadSafeData memoryMaxDataSeries = stage.getMemoryMaxDataSeries();
 
+    Range mTimeViewRangeUs = new Range(0, TimeUnit.SECONDS.toMicros(15));
```

Comment:
nit: don't prefix local vars with m. Also feel free to avoid prefixing fields with m as well. That is an ugly Java convention.